app/views/users/_private_message_button.html.erb

```diff
@@ -0,0 +1,3 @@
+<div>
+  <%= button_to "BarkMe",  conversations_path(recipient_id: user.id, sender_id: current_user.id), method: :post %>
```

Comment:
Bark me sounds a bit excessive ! 😆  we can change it later

Reply:
Yeah hahah! It's just to lough, it is not the final name